### PR TITLE
8259338: Add expiry exception for identrustdstx3 alias to VerifyCACerts.java test

### PR DESCRIPTION
**Pull request by RealCLanger**
identrustdstx3 is expiring soon. Test is failing in jdk11u since today. I see 11.0.13-oracle as well.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8259338](https://bugs.openjdk.java.net/browse/JDK-8259338): Add expiry exception for identrustdstx3 alias to VerifyCACerts.java test

### Reviewers
 * [Aleksey Shipilev](https://openjdk.java.net/census#shade) (@shipilev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk11u-dev pull/98/head:pull/98` \
`$ git checkout pull/98`

Update a local copy of the PR: \
`$ git checkout pull/98` \
`$ git pull https://git.openjdk.java.net/jdk11u-dev pull/98/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 98`

View PR using the GUI difftool: \
`$ git pr show -t 98`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk11u-dev/pull/98.diff">https://git.openjdk.java.net/jdk11u-dev/pull/98.diff</a>

</details>

### CODE REVIEW

**Review by shipilev — APPROVED** [review]
Looks good. The absence of this fix breaks tier1 :)

**Comment by RealCLanger** [comment]
> Looks good. The absence of this fix breaks tier1 :)

Yep. Waiting for bug approval by @jerboaa

**Pull request closed without merging**
